src/layouts/MainLayout.js

```diff
@@ -41,9 +42,21 @@ const MainLayout = ({ children, pageContext, sidebarOpen = true }) => {
     }
   };
 
+  /*
+   * [VSU] some docs pages are being designed as JS for faster experimenting
+   * and will never have the frontmatter property
+   * Using regex for check to account for paths with and without trailing slash
+   */
+  const docsAsJS = ['/introduction-apm'];
```

Comment:
did you mean this?
```suggestion
  const docsAsJS = [/introduction-apm/];
```
`String#match` will automatically try to convert its argument to a regex, so the string `'/introduction-apm'` is converted to `/\/introduction-apm/`

Reply:
Okay. I was wondering about these Regex nuances, but it was behaving as expected, so I just left it to the will of the PR gods. I'll make an update.